src/rx/rx_flysky_afhds.c

```diff
@@ -0,0 +1,111 @@
+#include "rx_flysky.h"
+
+#if defined(RX_FLYSKY)
+#include "drv_spi_a7105.h"
+#include <string.h>
+
+//------------------------------------------------------------------------------
```

Comment:
I can see how this may seem distracting, but when I develop various routines I like to delimit functions and provide function header comments of some sort when it makes sense. I'd like to retain these for now in case I need to get back to this code and perform maintenance. 

In the future I suppose we can remove them to maintain consistency with the rest of the codebase.